packages/@glimmer-workspace/integration-tests/test/keywords/each-test.ts

```diff
@@ -30,6 +31,39 @@ class Each extends RenderTest {
 
     this.assertHTML('  ');
   }
+
+  @test
+  'each with array of tags is reactive per item'(assert: Assert) {
+    let array = [createUpdatableTag(), createUpdatableTag()];
```

Comment:
I'm not sure we testing array here, because we dirty only items...
I think if we need to verify atomic array reactivity we may need to mutate array values by indexes and assert for amount of opcode operations in list-sync logic.